birdstagram/app/views/posts/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<h1> BIRDSTAGRAM </h1>
+<body>
+  <p>
+  <% if logged_in? %>
+    Hello, <%= current_user.username %>
+  <% end %>
+
+  <ul>
+    <% @posts.each do |post| %>
+      <li>
+        <div class = "post">
+        <b><%= link_to post.content, post %><br></b>
+        <% if post.image.attached? %>
```

Comment:
This is clear and I like that you're checking for an image first, then embedding it into the html image tag.